csharp/src/Ice/Communicator-PluginManager.cs

```diff
@@ -78,7 +78,7 @@ public string[] GetPlugins()
         {
             lock (_mutex)
             {
-                if (_state == StateDestroyed)
+                if (IsDisposed)
```

Comment:
I rename it to `CommunicatorDisposedException`

Reply:
We need a separate exception because we frequently catch and handle / ignore this exception?
Otherwise, I would just throw `ObjectDisposeException(typeof(Communicator).FullName)`.

Reply:
We ignore/handle it in a few places